pom.xml

```diff
@@ -19,31 +19,19 @@
     <packaging>pom</packaging>
 
     <modules>
-        <module>transaction-service-api</module>
         <module>transaction-service</module>
+        <module>transaction-service-api</module>
     </modules>
 
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
-
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-actuator</artifactId>
-        </dependency>
```

Comment:
Why was it removed and why it doesn't break anything?

Reply:
because it was useless 🤷‍♂️